src/Features/Core/Portable/UnsealClass/AbstractUnsealClassCodeFixProvider.cs

```diff
@@ -0,0 +1,88 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeActions;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Editing;
+using Microsoft.CodeAnalysis.FindSymbols;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.UnsealClass
+{
+    internal abstract class AbstractUnsealClassCodeFixProvider : CodeFixProvider
+    {
+        protected abstract string TitleFormat { get; }
+
+        public override FixAllProvider GetFixAllProvider()
+        {
+            // This code fix addresses a very specific compiler error. It's unlikely there will be more than 1 of them at a time.
+            return null;
+        }
+
+        public override async Task RegisterCodeFixesAsync(CodeFixContext context)
+        {
+            var syntaxRoot = await context.Document.GetSyntaxRootAsync(context.CancellationToken).ConfigureAwait(false);
+            var semanticModel = await context.Document.GetSemanticModelAsync(context.CancellationToken).ConfigureAwait(false);
```

Comment:
can you make locals for the parts of 'contxt' you repeatedly access.  i.e. 'context.Document' and 'context.CancellationToken'.

Reply:
➡️ Fixed in 644aaa3d2ca272f59e78587ada2bde5d281e493b